apollo-ios-codegen/Sources/IR/IR+EntitySelectionTree.swift

```diff
@@ -224,6 +224,23 @@ class EntitySelectionTree {
           targetSelections.mergeIn(scopeSelections, from: source)
         }
 
+        if let conditionalScopes = scopeConditions {
```

Comment:
Not sure why these changes are here? Not related to the work I am doing

Reply:
This is related to the fragment field merging work I'm doing. How the heck did this get here? Are you able to revert this back? I don't think this version is correct, and the one I'm working on is pretty heavily modified from this.